### ISSUE TITLE
[en] Incorrect comma? — 2023-04-29

### ISSUE THREAD

**Issue by marcoagpinto** [languagetool-org/languagetool#8369]
Hello!

The sentence:
`With the t-shirt I bought from the Great Jó Dos Theriormorphic years ago.`

It suggests a comma after “t-shirt”, but it doesn't sound very well.

Is it correct?

Thanks!

**Comment by BrianH824** [comment]
No, and a sentence that starts with "with" would typically be a compound sentence. This sentence is not, which is why it sounds a little funny. The trouble is that "I bought from the Great Jó Dos Theriormorphic years ago." is not an independent clause.

See: https://prowritingaid.com/can-you-start-a-sentence-with-with
